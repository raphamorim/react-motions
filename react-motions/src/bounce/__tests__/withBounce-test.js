import React from 'react';
import withBounce from '../withBounce';

describe('withBounce', () => {
  it('should render', () => {
    const wrapper = shallow(withBounce(<div>Bounce</div>));

    expect(wrapper).toMatchSnapshot();
  });
});
