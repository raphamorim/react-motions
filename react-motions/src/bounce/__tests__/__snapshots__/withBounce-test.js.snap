// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withBounce should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <div
    style={
      Object {
        "WebkitAnimationDuration": "1s",
        "WebkitAnimationFillMode": "both",
        "WebkitAnimationName": "bounce",
        "WebkitTransformOrigin": "center bottom",
        "animationDuration": "1s",
        "animationFillMode": "both",
        "animationName": "bounce",
        "transformOrigin": "center bottom",
      }
    }
  >
    <div>
      Bounce
    </div>
  </div>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div>
        Bounce
      </div>,
      "style": Object {
        "WebkitAnimationDuration": "1s",
        "WebkitAnimationFillMode": "both",
        "WebkitAnimationName": "bounce",
        "WebkitTransformOrigin": "center bottom",
        "animationDuration": "1s",
        "animationFillMode": "both",
        "animationName": "bounce",
        "transformOrigin": "center bottom",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Bounce",
      },
      "ref": null,
      "rendered": "Bounce",
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div>
          Bounce
        </div>,
        "style": Object {
          "WebkitAnimationDuration": "1s",
          "WebkitAnimationFillMode": "both",
          "WebkitAnimationName": "bounce",
          "WebkitTransformOrigin": "center bottom",
          "animationDuration": "1s",
          "animationFillMode": "both",
          "animationName": "bounce",
          "transformOrigin": "center bottom",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Bounce",
        },
        "ref": null,
        "rendered": "Bounce",
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
